### PR TITLE
Initial code for commerce/usageaggregates

### PR DESCRIPTION
**Pull request by Matt-Westphal**
Followed instructions: 
Generate the Node.js code. Run tools\UpdateNodeSdk.ps1 providing the target directory in your azure-sdk-for-node-pr (or azure-sdk-for-node) fork and the name of your service from SdkMappings.csv.  .\UpdateNodeSdk.ps1 C:\repos\azure-sdk-for-node-pr\lib\services <yourservice>  Now you have the code you can submit to the branch of azure-sdk-for-node-pr (or azure-sdk-for-node). 
And added the readme, package, and license files.  Not sure what else is needed for sure.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Matt-Westphal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by yugangw-msft** [comment]
any reason that you don't want to add factory method to create a client. also are you going to write cli commands using it? If not, then i suggest you don't make it to the official repo, as no one will use it

**Comment by Matt-Westphal** [comment]
First time working with Node.js, not sure what you mean by creating the client

I am planning to write cli commands.

**Comment by yugangw-msft** [comment]
i meant files like https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/resourceManagement/lib/resource.js 
It serves 2 goals:
1. Export needed objects so that client app can it
2. Convenient method to initialize the management client object

File: lib/services/commerce/lib/resource.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//
```
Comment (yugangw-msft) [inline lib/services/commerce/lib/resource.js]: rename this file to "commercs.js"

File: lib/services/commerce/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "description": "Microsoft Azure Commerce Usage Aggregates Management Client Library for Node",
+  "tags": [
+    "azure",
+    "sdk"
+  ],
+  "keywords": [
+    "node",
+    "azure"
+  ],
+  "main": "./lib/usageAggregationManagementClient.js",
```
Comment (yugangw-msft) [inline lib/services/commerce/package.json]: update to commerce.js as well

File: lib/services/commerce/lib/resource.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+exports.UsageAggregationManagementClient = UsageAggregationManagementClient;
+
+/**
+* Creates a new {@link UsageAggregationManagementClient} object.
+*
+* @param {string} [credentials.subscriptionId]      The subscription identifier.
+* @param {string} [baseUri]                         The base uri.
+* @return {UsageAggregationManagementClient}        A new UsageAggregationManagementClient object.
+*/
+exports.createUsageAggregationManagementClient = function (credentials, baseUri) {
```
Comment (yugangw-msft) [inline lib/services/commerce/lib/resource.js]: expose the filter parameter

**Pull request merged**